src/components/grid-aware/ImageHeader/ImageHeader.module.css

```diff
@@ -0,0 +1,125 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-column-start: 2;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-top: 30px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaLeft {
+  grid-column: 2 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaRight {
+  grid-column: 9 / span 4;
+  grid-row: 1;
+}
+
+
+/* Left grid area */
+
+.titleWrapper {
+  font: var(--font-headline);
+  margin-top: 90px;
+}
+
+.subtitleWrapper {
+  font: var(--font-body-large);
+  margin-top: 15px;
+}
+
+.descriptionWrapper {
+  font: var(--font-body-medium);
+  margin-top: 20px;
+}
+
+.ctaButtonWrapper {
+  margin-bottom: 83px;
+  margin-top: 30px;
+}
+
+@media (--tablet-and-down) {
+  .titleWrapper {
+    margin-top: 30px;
+  }
+
+  .ctaButtonWrapper {
+    margin-bottom: 0;
+  }
+}
+
+
+/* Right grid area*/
+
+.image {
+  height: auto;
+  max-width: 100%;
+}
+
+.image1Wrapper {
+  margin-top: 50px;
+  position: relative;
+  z-index: 1;
```

Comment:
The use of z-index is discouraged. If a sticky navbar is used, z-index would have to take into account items such as that.